solidity/dashboard/src/services/tbtc-authorization.service.js

```diff
@@ -190,18 +205,20 @@ const fetchBondingData = async (web3Context) => {
     for (let i = 0; i < operators.length; i++) {
```

Comment:
Also, with operators as a Map, iteration through operators could be cleaner. We could: `for (const [operatorAddress, isWithdrawable] of operators.entries())`. By using Map, we can avoid unclear variables like `operators[i][0]`.

Reply:
yea, I like that!